datastore/additional_data/models.py

```diff
@@ -156,3 +156,17 @@ class Meta:
 
     def __str__(self):
         return self.tsg_org_type
+
+
+class CodeList(models.Model):
```

Comment:
I'm not sure the name of this makes sense, as this is not a codelist (e.g. the regrantType codelist), but rather an individual codelist code/item.

Reply:
Agree, I couldn't think of an alternative at the time, maybe CodelistCode works?

Reply:
Thanks, Now changed to CodelistCode